app/experimenter/experiments/tests/factories.py

```diff
@@ -171,6 +171,18 @@ def create_with_status(cls, target_status, *args, **kwargs):
             old_status = status_value
             now += datetime.timedelta(days=random.randint(5, 20))
 
+        # set signoffs to true
+        if experiment.status in (
+            Experiment.STATUS_SHIP,
+            Experiment.STATUS_ACCEPTED,
+            Experiment.STATUS_LIVE,
+            Experiment.STATUS_COMPLETE,
+        ):
```

Comment:
Can you move this into an experiments model method called `is_shipped`

Reply:
will do